### DISCUSSION TITLE
Stuck: Installation process loops back to Recovery menu

### DISCUSSION THREAD

**Discussion by RudiKroch** [valnoxy/t480-oc#151] [General]
Hi everyone,

I'm getting stuck and can't really find any information on how to proceed from here, so I'm hoping someone can offer me advice.

I believe I followed the instructions to the letter, and all goes well until I select "Reinstall macOS" from the recovery screen. At stage, the installation proceeds, seems to complete (takes ages), and the Laptop restarts. Then, I get kicked back into Recovery with all the same options, as if nothing happened. I selected "Reinstall macOS" again, but the process just seemed to repeat (taking a few hours again).

I'm kinda stumped, since I can find no information here or elsewhere of this happening to anyone else. Could you please give me some ideas of what could be going on?

I have a T480 with a i7 8650U.

Appreciate it!

**Comment by khlivnyi** [comment]
Disable secure boot before installing, and everything will be fine. Also, if WiFi will not work, install beta airport for Sonoma from the official GitHub page. Cheers.

**Reply by RudiKroch** [reply]
Thanks for your reply, @khlivnyi! Secure boot was disabled before I installed - I'm literally looking at the BIOS right now to double-check, haha.

By the way, I have the laptop hooked up via ethernet. In case you're wondering, yes, I added the MAC address of my ethernet device in the PlatformInfo -> Generic -> ROM field of the config.plist.

**Comment by khlivnyi** [comment]
Secure boot model in plist.conf I mean

**Comment by RudiKroch** [comment]
Hmmm...nope, still the same behaviour. For reference, I changed the SecureBootModel field under Misc.

However, few questions:

1. The instructions say to enable/disable various ACPI patches. I took "disable" to mean I should delete the appropriate .aml files in the ...EFI\OC\ACPI directory. Is that the correct way to do it?
2. In valnoxy's [video tutorial](https://www.youtube.com/watch?v=thYDWyJuUq4) (which I just discovered), at 8:45, the boot menu shows Opencore as an option. That's not present when I get to this; I only have Windows Boot Manager, NVMe0, USB HDD, and PCIE LAN. Should I have Opencore as an option? For my installation, I selected the USB HDD which contains the EFI partition.

Thanks again, I appreciate it.
